setup.py

```diff
@@ -51,7 +52,7 @@
     keywords='xova',
     name='xova',
     packages=find_packages(),
-    python_requires=">=3.6",
+    python_requires=">=3.7",
```

Comment:
Daskms version for 3.6 does not have high enough release version. Think we can safely drop support for 3.6 now. Tests migrated to 3.7 and 3.8.

Reply:
In fact, dask-ms has migrated to 3.8, so we might want to move this to 3.8 too.

Reply:
Hmm ok I think we should move it once we get in new changes to read out keywords (that will be a new daskms release) or if there is a bugfix in the bda/averaging code? For the moment both versions seems to be working